src/SFA.DAS.Apprenticeships.Web/Views/Service/SignedOut.cshtml

```diff
@@ -0,0 +1,17 @@
+@model SignedOutViewModel
```

Comment:
On provider sign out we redirect to the main provider app, should this not follow the same behavior?